pkgs/applications/science/physics/elmerfem/default.nix

```diff
@@ -1,68 +1,78 @@
-{ lib, stdenv, fetchFromGitHub, cmake, git, gfortran, mpi, blas, liblapack, pkg-config, libGL, libGLU, opencascade, libsForQt5, vtkWithQt5}:
+{ blas
+, cmake
+, fetchFromGitHub
+, gfortran
+, git
+, lib
+, liblapack
+, libsForQt5
+, libGL
+, libGLU
+, mpi
+, opencascade
+, pkg-config
+, stdenv
+, vtkWithQt5
+}:
 
 stdenv.mkDerivation rec {
   pname = "elmerfem";
-  version = "9.0";
+  version = "unstable-2022-08-30";
 
   src = fetchFromGitHub {
     owner = "elmercsc";
     repo = "elmerfem";
-    rev = "release-${version}";
-    sha256 = "VK7jvu4s5d7k0c39XqY9dEzg/vXtX5Yr/09VcuZVQ9A=";
+    rev = "3eafebb4a0287c435376207e15003bf32ffb11e3";
+    hash = "sha256-2lVJmbFBDnUo+bw/AILzUncOJM4B7eAW0mosmsAWANI=";
   };
 
+  storepath = placeholder "out";
+
+  cmakeFlags = [
+    "-DELMER_INSTALL_LIB_DIR=${storepath}/lib"
+    "-DWITH_OpenMP:BOOLEAN=TRUE"
+    "-DWITH_MPI:BOOLEAN=TRUE"
+    "-DWITH_QT5:BOOLEAN=TRUE"
+    "-DWITH_OCC:BOOLEAN=TRUE"
+    "-DWITH_VTK:BOOLEAN=TRUE"
+    "-DWITH_ELMERGUI:BOOLEAN=TRUE"
+    "-DCMAKE_INSTALL_LIBDIR=lib"
+    "-DCMAKE_INSTALL_INCLUDEDIR=include"
+    "-DCMAKE_OpenGL_GL_PREFERENCE=GLVND"
+  ];
+
+  preConfigure = ''
+    patchShebangs ./
```

Comment:
Can we be more specific?

Reply:
apparently not

Reply:
You are sure? Can we at least specific a directory?

Reply:
naah. i dropped the whole thing and the build still works. it seems to be unneccessary.